TeamCode/src/main/java/org/firstinspires/ftc/teamcode/localization/ThreeDeadWheelLocalizer.java

```diff
@@ -48,9 +50,28 @@ public ThreeDeadWheelLocalizer(HardwareMap hardwareMap, double inPerTick) {
         this.inPerTick = inPerTick;
 
         FlightRecorder.write("THREE_DEAD_WHEEL_PARAMS", PARAMS);
+
+        this.pose = pose;
+    }
+
+    @Override
+    public void setPose(Pose2d pose) {
+        this.pose = pose;
+    }
+
+    @Override
+    public Pose2d getPose() {
+        return pose;
+    }
+
+    @Override
+    public PoseVelocity2d update() {
+        Twist2dDual<Time> delta = calculatePoseDelta();
+        pose = pose.plus(delta.value());
+        return delta.velocity().value();
     }
 
-    public Twist2dDual<Time> update() {
+    public Twist2dDual<Time> calculatePoseDelta() {
```

Comment:
Can you fold this into the `update()` method?

Reply:
done